docs/topics/forms/descriptions/connect-description.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Connect the description
+layout: default
+parent: Input and description
+nav_order: 2
+---
+
+# Connect the description to the form field
+
+For screen reader users, it's important that the description is announced along with the form field. By connecting the description to the form field using `aria-describedby`, the label and description are announced together when a screen reader user focuses on the form field.
+
+## Why is this important?
+
+Screen readers, such as JAWS, switch to "forms mode" when processing content within a `<form>` element. In this mode, screen readers only read the form fields, including any associated information (for example with `aria-describedby`). Unassociated information is not read aloud unless the user searches for it.
```

Comment:
Is "such as JAWS" necessary here? All screen readers do this, and I'm not sure extra clause helps. Useful in the context of an isolated article, but we'd be using the same construction over and over on this site, and I think it would be unnecessarily repetitive.

Reply:
@joedolson, thsnkf for reviewing.

> Is "such as JAWS" necessary here?
VoiceOver doens't, but I agree that it is incomplete. I changed the sentence into "Screen readers can switch to "forms mode" to make it more generic. Because it's also depending on the settings.